app/lib/server/server.go

```diff
@@ -228,6 +228,7 @@ func CheckServers(g *config.Group) error {
 	}
 
 	if !success {
+		log.Error("at least one failed")
```

Comment:
Why logging ? If it's already printed at the screen for each failing line + a returning error at the end of this function...